core/object/object.h

```diff
@@ -118,8 +118,10 @@ enum PropertyUsageFlags {
 	PROPERTY_USAGE_EDITOR_INSTANTIATE_OBJECT = 1 << 26, // For Object properties, instantiate them when creating in editor.
 	PROPERTY_USAGE_EDITOR_BASIC_SETTING = 1 << 27, //for project or editor settings, show when basic settings are selected.
 	PROPERTY_USAGE_READ_ONLY = 1 << 28, // Mark a property as read-only in the inspector.
+	PROPERTY_USAGE_SECRET = 1 << 29, // Export preset credentials that should be stored separately from the rest of the export config.
 
 	PROPERTY_USAGE_DEFAULT = PROPERTY_USAGE_STORAGE | PROPERTY_USAGE_EDITOR,
+	PROPERTY_USAGE_CREDENTIALS = PROPERTY_USAGE_DEFAULT | PROPERTY_USAGE_SECRET,
```

Comment:
I wouldn't add this, it's not used often enough to warrant a dedicated value. You can use `PROPERTY_USAGE_DEFAULT | PROPERTY_USAGE_SECRET` directly, as done for most of the other 30-ish usage flags.